set-postgres-permissions.bash

```diff
@@ -41,16 +24,37 @@ export PGPASSWORD=$DB_PASSWORD
 
 JENKINS_SQL_COMMAND="
 GRANT ALL ON ALL TABLES IN SCHEMA public TO \"${DB_USER}\";
+GRANT ${DB_ADMIN} to \"${DB_USER}\";
+GRANT ${DB_ADMIN} to \"${DB_ADMIN_GROUP}\";
```

Comment:
extra grants needed so that admin users have permission outside of the `postgres` database